release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -197,6 +197,13 @@ For more information about assigning IP addresses from an IP address pool to ser
 [id="ocp-4-13-osdk"]
 === Operator development
 
+[id="ocp-4-13-suggested-namespace-template"]
+==== Setting a suggested namespace with default node selector
```

Comment:
I would suggest using 'suggested-namespace-template'
The base Story for this part can be tracked on https://issues.redhat.com/browse/CONSOLE-3120 

As we already have an annotation called 'operatorframework.io/suggested-namespace' in the operator CSV yaml file. So this description may bring some misunderstanding.
The newly released annotation is called operatorframework.io/**suggested-namespace-template** which needs to be distinguished